src/style.css

```
@@ -0,0 +1,42 @@
+/* Reset browser's default margin */
+.body {
+  margin: 0;
+}
+/* stars block in column */
+.stars {
+  display: flex;
+}
+
+/* each star decoration without active stars */
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+/* active stars*/
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3), 
.stars--4 .stars__star:nth-child(-n+4), 
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(images/star-active.svg);
}
```
You have one property for some classes, combine selectors